packages/edit-post/src/components/sidebar/settings-sidebar/style.scss

```diff
@@ -1,12 +1,13 @@
 .edit-post-settings-sidebar__panel-block .components-panel__body {
 	border: none;
 	border-top: $border-width solid $light-gray-500;
-	margin: 0 -16px;
+	margin: 0 -$grid-size;
```

Comment:
This should probably be `$grid-size-large`, as `$grid-size` is only 8px. That's probably what's causing the separators to not go edge to edge.